_sass/components/_card.scss

```diff
@@ -121,6 +125,12 @@ img.card-image {
     font-weight: $weight-bold;
     margin-bottom: $base-padding-lite;
   }
+
+  @include respond-to(medium-up) {
```

Comment:
Expected item on line 129 to appear before line 117. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets